homeassistant/components/light/depict.py

```diff
@@ -0,0 +1,96 @@
+"""
```

Comment:
We've updated where we store platforms. Please move this file to `depict/light.py`. Content doesn't need a change for this rename, HA can pick it up. Make sure you rebase on latest dev.